ghga_event_schemas/pydantic_.py

```diff
@@ -260,6 +260,30 @@ class Config:
         title = "file_download_served"
 
 
+class EmailNotification(BaseModel):
+    """This event is emitted by services that desire to send an email notification.
+    It is picked up by the notification service."""
+
+    to_: EmailStr = Field(..., description="The primary recipient of the email")
```

Comment:
Why the underscore, pylint thinking this is to short?
I'd rather keep is at `to` and mute the warning.

According to https://docs.pydantic.dev/usage/types/ `EmailStr` needs https://github.com/JoshData/python-email-validator. Is this pulled in somewhere or covered by the pydantic[email] dependency?

Reply:
I had a moment where I thought to was a reserved word. I changed the to_ to to.
And yes, the [email] pulls in the email-validator. See https://docs.pydantic.dev/install/